### PR TITLE
Fix flexberry-checkbox & update semantic-ui-ember addon

### PR DESCRIPTION
**Pull request by yzaripova**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "ember-browserify": "1.1.9",
     "dexie": "^1.3.6",
     "ember-localforage-adapter": "git+https://github.com/Flexberry/ember-localforage-adapter.git",
-    "semantic-ui-ember": "0.9.3"
+    "semantic-ui-ember": "2.0.0-beta.0"
```
Comment (KollegOFF) [inline package.json]: I was plagued by doubts, i think it will be max pain.

Comment (yzaripova) [inline package.json]: I don't know. It does not update the Ember.js, just semantic addon.

Comment (KollegOFF) [inline package.json]: But `flexberry-dropdown` getting crazy.

Comment (ehaberev) [inline package.json]: Probably problems with dropdown were fixed by @DubrovinPavel in #181

Comment (yzaripova) [inline package.json]: I forgot update changelog :(

**Pull request merged**